README.md

```diff
@@ -24,47 +24,42 @@ and Java (Android and regular Java), across Windows, Linux and macOS platforms.
 
 ## Getting Started
 
-Check out our [documentation to get started](https://mono.github.io/Embeddinator-4000/)
+Check out our [documentation to get started](https://mono.github.io/Embeddinator-4000/).
 
 ## Community
 
 Feel free to join us at our [#managed-interop](https://gitter.im/managed-interop) Gitter discussion channel.
 
 ## Building
 
-Clone this repository and initialize/update submodules as well as solution depends on them.
+- Clone this repository 
+- Initialize/update submodules: `git submodule update --recursive --init`
+- Open the solution file `Embeddinator-4000.sln` with Visual Studio or Xamarin Studio 
```

Comment:
Should we still mention Xamarin Studio? I think we want Visual Studio for Mac instead